<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alex Russkikh</author>
 <class>wellsMarkDialog_base</class>
 <widget class="QDialog" name="wellsMarkDialog_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark wells</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Текст</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsFieldExpressionWidget" name="mFieldExpMarkText" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Цвет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsColorButtonV2" name="mCBtnLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Заливка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsColorButtonV2" name="mCBtnFill"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBoxClearOldMark">
     <property name="text">
      <string>Очистить текущие метки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOpenBrowser">
     <property name="text">
      <string>Выбрать скважины</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>wellsMarkDialog_base</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>wellsMarkDialog_base</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenBrowser</sender>
   <signal>clicked()</signal>
   <receiver>wellsMarkDialog_base</receiver>
   <slot>on_btnOpenBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnOpenBrowser()</slot>
 </slots>
</ui>
